businesscentral/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    information:
+        - name: "JBOSS_PRODUCT"
+          value: "bpmsuite"
+        - name: "JBOSS_BPMSUITE_VERSION"
+          value: "7.0.0.CAP01"
```

Comment:
Isn't there a way to figure it out? Generally speaking - we should not use DR's, CAP's or anything like this in version. Can you tell me what version is advertised on boot?